spring-cloud-dataflow-shell/pom.xml

```diff
@@ -22,7 +22,7 @@
 		<dependency>
 			<groupId>org.springframework.shell</groupId>
 			<artifactId>spring-shell</artifactId>
-			<version>1.1.0.RELEASE</version>
+			<version>1.2.0.BUILD-SNAPSHOT</version>
```

Comment:
I hope we are planning to have a release from `spring-shell` before dataflow milestone release.